crypto/bn254/signer.go

```diff
@@ -0,0 +1,9 @@
+package bn254
+
+import "context"
+
+type Signer interface {
+	SignMessage(ctx context.Context, message [32]byte) ([]byte, error)
+
+	GetOperatorId() (string, error)
```

Comment:
should there be `context.Context` here? it's a pure operation - as in it doesn't depend on network in any case.

Reply:
Then no. Unless you think some implementation in the future might need to make this call over the network, for eg calling a contract?

Reply:
wait how does cerberus get operatorId? Doesn't it have to ask the remote signer over the network?

Reply:
operatorId is just extracted out of public key. so that call just not need to go over the network since public key is provided by the caller when initializing the client.